lib/linear_gauge/linear_gauge_label.dart

```diff
@@ -78,9 +86,22 @@ class LinearGaugeLabel {
               b.dx * (i / (totalRulers + 1));
           double y = a.dy * (1 - ((i) / (totalRulers + 1))) +
               b.dy * (i / (totalRulers + 1));
+
           if (Offset(x, y) != a) {
-            canvas.drawLine(
-                Offset(x, y), Offset(x, 5 + height), secondaryRulersPaint);
+            if (inverted) {
+              canvas.drawLine(
```

Comment:
Can you calculate only offsets and use `canvad.drawLine` once

Reply:
`
if (Offset(x, y) != a) {

            // the co-ordinate points where secondary ruler will end
            Offset secondaryRulerEndPoint;
            if (inverted) {
              //the value 5 for the offset y axis is the height parameter for the secondary rulers
              secondaryRulerEndPoint =
                  Offset(x, -(5 + height - linearGaugeHeight));
            } else {
              //the value 5 for the offset y axis is the height parameter for the secondary rulers
              secondaryRulerEndPoint = Offset(x, 5 + height);
            }
            canvas.drawLine(
              Offset(x, y),
              secondaryRulerEndPoint,
              secondaryRulersPaint,
            );

          }`